### PR TITLE
Adding support for writing EQN param in edif file. Also, removing sup…

### PR DESCRIPTION
**Pull request by lnsharma**
…port to write INIT param for DFFs

### CODE REVIEW

File: ql-qlf-plugin/Makefile [hunk 1]
```diff
@@ -9,7 +9,8 @@
 NAME = ql-qlf
 SOURCES = synth_quicklogic.cc \
           ql-dsp.cc \
-          pp3_braminit.cc
+          pp3_braminit.cc \
+	  quicklogic_eqn.cc
```
Comment (acomodi) [inline ql-qlf-plugin/Makefile:13]: This seems to be misaligned with the rest

**Review by acomodi — CHANGES_REQUESTED** [review]
There is one minor formatting issue, but otherwise LGTM

File: ql-qlf-plugin/Makefile (hunk 1 above)
Comment (lnsharma) [reply ql-qlf-plugin/Makefile]: Corrected it

File: ql-qlf-plugin/Makefile [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 #
-# Use of this source code is governed by a ISC-style
-# license that can be found in the LICENSE file or at
-# https://opensource.org/licenses/ISC
+#Use of this source code is governed by a ISC - style
+#license that can be found in the LICENSE file or at
+#https: // opensource.org/licenses/ISC
 #
-# SPDX-License-Identifier:ISC
+#SPDX - License - Identifier : ISC
```
Comment (acomodi) [inline ql-qlf-plugin/Makefile:7]: I think this was changed by the clang formatter. This change should be reverted

**Review by acomodi — APPROVED** [review]
LGTM, merging

**Pull request merged**